h2o-py/h2o/frame.py

```diff
@@ -3941,19 +3944,19 @@ def convert_H2OFrame_2_DMatrix(self, predictors, yresp, h2oXGBoostModel, return_
         Follow the steps below to compare H2OXGBoost and native XGBoost:
 
         1. Train the H2OXGBoost model with H2OFrame trainFile and generate a prediction:
-        h2oModelD = H2OXGBoostEstimator(**h2oParamsD) # parameters specified as a dict()
-        h2oModelD.train(x=myX, y=y, training_frame=trainFile) # train with H2OFrame trainFile
-        h2oPredict = h2oPredictD = h2oModelD.predict(trainFile)
+        ``h2oModelD = H2OXGBoostEstimator(**h2oParamsD) # parameters specified as a dict()``
+        ``h2oModelD.train(x=myX, y=y, training_frame=trainFile) # train with H2OFrame trainFile``
+        ``h2oPredict = h2oPredictD = h2oModelD.predict(trainFile)``
 
         2. Derive the DMatrix from H2OFrame:
-        nativeDMatrix = trainFile.convert_H2OFrame_2_DMatrix(myX, y, h2oModelD)
+        ``nativeDMatrix = trainFile.convert_H2OFrame_2_DMatrix(myX, y, h2oModelD)``
 
         3. Derive the parameters for native XGBoost:
-        nativeParams = h2oModelD.convert_H2OXGBoostParams_2_XGBoostParams()
+        ``nativeParams = h2oModelD.convert_H2OXGBoostParams_2_XGBoostParams()``
 
         4. Train your native XGBoost model and generate a prediction:
-        nativeModel = xgb.train(params=nativeParams[0], dtrain=nativeDMatrix, num_boost_round=nativeParams[1])
-        nativePredict = nativeModel.predict(data=nativeDMatrix, ntree_limit=nativeParams[1].
+        ``nativeModel = xgb.train(params=nativeParams[0], dtrain=nativeDMatrix, num_boost_round=nativeParams[1])``
+        ``nativePredict = nativeModel.predict(data=nativeDMatrix, ntree_limit=nativeParams[1].``
```

Comment:
What about a single code block for this:
```suggestion
        Follow the steps below to compare H2OXGBoost and native XGBoost::
        
            # 1. Train the H2OXGBoost model with H2OFrame trainFile and generate a prediction:
            h2oModelD = H2OXGBoostEstimator(**h2oParamsD) # parameters specified as a dict()
            h2oModelD.train(x=myX, y=y, training_frame=trainFile) # train with H2OFrame trainFile
            h2oPredict = h2oPredictD = h2oModelD.predict(trainFile)

            # 2. Derive the DMatrix from H2OFrame:
            nativeDMatrix = trainFile.convert_H2OFrame_2_DMatrix(myX, y, h2oModelD)

            # 3. Derive the parameters for native XGBoost:
            nativeParams = h2oModelD.convert_H2OXGBoostParams_2_XGBoostParams()

            # 4. Train your native XGBoost model and generate a prediction:
            nativeModel = xgb.train(params=nativeParams[0], dtrain=nativeDMatrix, num_boost_round=nativeParams[1])
            nativePredict = nativeModel.predict(data=nativeDMatrix, ntree_limit=nativeParams[1].

            # 5. Compare the predictions h2oPredict from H2OXGBoost, nativePredict from native XGBoost.
```
I find the result more appealing visually and users can copy paste it integrally in a Jupyter notebook for example.